README.md

```diff
@@ -0,0 +1,32 @@
+# KTorrent (org.kde.ktorrent)
+
+[Homepage](https://kde.org/applications/ktorrent) |
+[User documentation](https://docs.kde.org/stable5/en/extragear-network/ktorrent/index.html) |
+[Sources](https://invent.kde.org/network/ktorrent)
+
+## Reporting bugs
+
+**Before filling a bug, please test if the bug is reproduceable with the
+classic non-Flatpak version of this application.**
+
+- If the bug is only reproducible with the **Flatpak version** of this
+  application, please file an [Issue here][issue].
+
+- If the bug is also reproducible with the **non-Flatpak version** of this
+  application, please take a look at the [currently opened bugs][bugs] and if
+  it has not been reported yet, file a bug at
+  [bugs.kde.org](https://bugs.kde.org).
+
+**If you have any doubt, please file an [Issue here][issue].**
+
+## Permissions rationale
```

Comment:
I would only document the non-obvious ones (`--filesystem=host`) inside the manifest and drop the rest.

Reply:
Yeah, you only need to document anything outside the unrestricted permissions - and even then only *really* in this review. https://github.com/flathub/flathub/wiki/App-Requirements#unrestricted-permissions

Reply:
I've kept only the restricted ones. I would prefer that we keep those explanations in the README to make them easier to discover for potential contributors or users looking at the repo (and the json file format does not really support comments). Maybe we could switch to in-manifest comments once we move to yaml.

Reply:
"Unrestricted permissions

The following permissions can be freely used:"

Unrestricted by what? who? is this a flatpak policy or somehow flathub has created their own policy on top of permissions? I read those two sentences and i get no understanding of what they really mean. 

Can someone please clarify?